omega/omega-context/src/main/java/org/apache/servicecomb/pack/omega/context/TransactionContextProperties.java

```diff
@@ -17,9 +17,11 @@
 package org.apache.servicecomb.pack.omega.context;
 
 /**
- * Once the user business class implement this TransactionContextWrapper, Omega could extract the TransactionContext instance
- * out of the business class, and set up the OmegaContext before calling sub transaction method.
+ * Once the user business class implement this TransactionContextWrapper, Omega could extract the GlobalTransactionId
+ * and LocalTransactionId out of the business class, and set up the OmegaContext before calling sub transaction method.
  */
-public interface TransactionContextWrapper {
-  TransactionContext getTransactionContext();
+public interface TransactionContextProperties {
+  String getGloableTxId();
```

Comment:
Typo, should be `getGlobalTxId()`

Reply:
sorry，it's my mistake. I just submit a quick fix for these typos.